### ISSUE TITLE
No alt text for any of the buttons and objects

### ISSUE THREAD

**Issue by chad3814** [spqr-game/spqr-issues#28]
There should be alt text on every element of the game, this helps with mouse overs, screen readers and overall understanding what each button does
